src/UI/Pages/Home/CardView.tsx

```diff
@@ -78,28 +77,32 @@ const EnvironmentCard: React.FC<EnvironmentCardProps> = ({
 }) => (
   <Card isHoverable isCompact aria-label={"Environment card"}>
     <CardHeader>
-      <CardTitle>{environment.name}</CardTitle>
+      {environment.icon && (
+        <img
+          style={{ height: "2em" }}
+          src={`data:${environment.icon}`}
+          alt={words("home.environment.icon")(environment.name)}
+        ></img>
+      )}
```

Comment:
`img` can be a self closing element I think.
And we already have some styled components in this file.
Might as well make the image a styled component as well.